testnet/lighthouse/lighthouse.toml

```diff
@@ -0,0 +1,21 @@
+testnet-dir = "./configs/testnet/shared"
+datadir = "./testnet-data/consensus"
+execution-jwt = "./configs/shared/secrets/jwt.hex"
+execution-endpoint = "http://localhost:8551"
+subscribe-all-subnets = true
+listen-address = "0.0.0.0"
+eth1 = true
+http = true
+http-address = "0.0.0.0"
+http-port = 4000
+enr-udp-port = 9000
+enr-tcp-port = 9000
+discovery-port = 9000
+enable-private-discovery = true
+enr-address = "0.0.0.0"
+metrics = true
+metrics-address = "0.0.0.0"
+metrics-allow-origin = "*"
+metrics-port = 5054
+boot-nodes = "enr:-MK4QL_QJUpcwZszA99XzR9nDL9FiXmt_asTqrpCy4wtDzRzIJvSAsan9LI3umpDKWTOpvEMuOdg0feAgXeaG-5C3uSGAYdLen_mh2F0dG5ldHOIAAAAAAAAAACEZXRoMpBy3Hm5QgEABP__________gmlkgnY0gmlwhCJaUYyJc2VjcDI1NmsxoQKZw1caFRxUwB-ee7gSDc1ZGqr13ngROXHEokVKNbvYuIhzeW5jbmV0cwCDdGNwgjLIg3VkcIIu4A"
+slots-per-restore-point = 32
```

Comment:
We can remove it, this is relevant for explorers only.

Reply:
Done